lib/src/executor/host.rs

```diff
@@ -1702,21 +1719,85 @@ impl ReadyToRun {
                     key_ptr,
                     key_size,
                     value: None,
+                    old_value: None,
                     inner: self.inner,
                 })
             }
-            HostFunction::ext_offchain_is_validator_version_1 => host_fn_not_implemented!(),
-            HostFunction::ext_offchain_submit_transaction_version_1 => host_fn_not_implemented!(),
-            HostFunction::ext_offchain_network_state_version_1 => host_fn_not_implemented!(),
-            HostFunction::ext_offchain_timestamp_version_1 => host_fn_not_implemented!(),
-            HostFunction::ext_offchain_sleep_until_version_1 => host_fn_not_implemented!(),
-            HostFunction::ext_offchain_random_seed_version_1 => host_fn_not_implemented!(),
-            HostFunction::ext_offchain_local_storage_set_version_1 => host_fn_not_implemented!(),
-            HostFunction::ext_offchain_local_storage_compare_and_set_version_1 => {
+            HostFunction::ext_offchain_is_validator_version_1 => HostVm::ReadyToRun(ReadyToRun {
+                inner: self.inner,
+                resume_value: Some(vm::WasmValue::I32(1)),
```

Comment:
```suggestion
                resume_value: Some(vm::WasmValue::I32(1)),   // TODO: ask the API user
```

I'm fine with partially-implemented stuff, but I would like TODOs to indicate where things are not finished.